docs/src/main/asciidoc/context-propagation.adoc

```diff
@@ -10,7 +10,7 @@ include::./attributes.adoc[]
 Traditional blocking code uses link:https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/ThreadLocal.html[`ThreadLocal`]
  variables to store contextual objects in order to avoid
 passing them as parameters everywhere. Many Quarkus extensions require those contextual objects to operate
-properly: link:getting-started.html[RESTEasy], link:getting-started.html[ArC] and link:transaction.html[Transaction]
+properly: link:rest-json[RESTEasy], link:cdi-reference[ArC] and link:transaction[Transaction]
```

Comment:
ah, this one was wrong ?!

Reply:
Yes, they were pointing to the getting started guide. Better point to the guides of each component.

I fixed another one too, it was pointing to an old `async-message-passing` link (we have a redirection - with a canonical meta!) but better avoid it.